### ISSUE TITLE
Sharing file leads to freeze browser window (focus-trap issue)

### ISSUE THREAD

**Issue by fabricemous** [nextcloud/server#35955]
When sharing a file from within the file the browser window freezes. When sharing via file management you can work around this. 

**Browser**

Using Chrome browser (up to date) Version 108.0.5359.124 (Official Build) (arm64) on MacOS Ventura (13.0.1 (22A400)). I cannot confirm but colleague is experiencing the same on Windows using Chrome and Edge 

**Log file** 
I attached output from console log to this issue. I hope this gives some clue. 

```
11:44:29.782 preview:1          Failed to load resource: the server responded with a status of 404 ()
11:44:29.783 preview:1          Failed to load resource: the server responded with a status of 404 ()
11:44:29.783 globals.js:60 The select2 library is deprecated! It will be removed in nextcloud 19.
tn @ globals.js:60
11:44:29.784 globals.js:60 The select2 library is deprecated! It will be removed in nextcloud 19.
tn @ globals.js:60
11:44:29.788 preview:1          Failed to load resource: the server responded with a status of 404 ()
11:44:29.788 preview:1          Failed to load resource: the server responded with a status of 404 ()
11:44:29.788 preview:1          Failed to load resource: the server responded with a status of 404 ()
11:44:29.837 preview:1          Failed to load resource: the server responded with a status of 404 ()
11:44:29.837 preview:1          Failed to load resource: the server responded with a status of 404 ()
11:44:29.837 preview:1          Failed to load resource: the server responded with a status of 404 ()
11:44:29.915 preview:1          Failed to load resource: the server responded with a status of 404 ()
11:44:29.915 preview:1          Failed to load resource: the server responded with a status of 404 ()
11:44:29.915 preview:1          Failed to load resource: the server responded with a status of 404 ()
11:44:29.915 preview:1          Failed to load resource: the server responded with a status of 404 ()
11:44:29.917 preview:1          Failed to load resource: the server responded with a status of 404 ()
11:44:29.917 preview:1          Failed to load resource: the server responded with a status of 404 ()
11:44:29.918 preview:1          Failed to load resource: the server responded with a status of 404 ()
11:44:29.987 capabilities.js:35 OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities
getCapabilities @ capabilities.js:35
11:44:29.987 capabilities.js:35 OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities
getCapabilities @ capabilities.js:35
11:44:29.988 capabilities.js:35 OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities
getCapabilities @ capabilities.js:35
11:44:29.988 capabilities.js:35 OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities
getCapabilities @ capabilities.js:35
11:44:29.988 capabilities.js:35 OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities
getCapabilities @ capabilities.js:35
11:44:29.988 SharingInput.vue:321 recommendations Array(6)
11:44:34.265 DevTools failed to load source map: Could not load content for chrome-extension://cfhdojbkjhnklbpkdaibdccddilifddb/browser-polyfill.js.map: System error: net::ERR_FILE_NOT_FOUND
11:44:37.087 [Violation] Forced reflow while executing JavaScript took 47ms
11:45:08.811 DevTools failed to load source map: Could not load content for chrome-extension://cfhdojbkjhnklbpkdaibdccddilifddb/browser-polyfill.js.map: System error: net::ERR_FILE_NOT_FOUND
11:45:08.888 focus-trap.esm.js:342 Uncaught 
e @ focus-trap.esm.js:342
y @ focus-trap.esm.js:407
e @ focus-trap.esm.js:342
y @ focus-trap.esm.js:407
e @ focus-trap.esm.js:342
y @ focus-trap.esm.js:407
e @ focus-trap.esm.js:342
…
…
…
…
y @ focus-trap.esm.js:407
e @ focus-trap.esm.js:342
(anonymous) @ focus-trap.esm.js:538
setTimeout (async)
l @ focus-trap.esm.js:80
E @ focus-trap.esm.js:537
r @ focus-trap.esm.js:602
activate @ focus-trap.esm.js:611
(anonymous) @ NcActions.js:2
u @ runtime.js:63
(anonymous) @ runtime.js:294
(anonymous) @ runtime.js:119
o @ NcActions.js:2
s @ NcActions.js:2
Promise.then (async)
o @ NcActions.js:2
s @ NcActions.js:2
(anonymous) @ NcActions.js:2
(anonymous) @ NcActions.js:2
useFocusTrap @ NcActions.js:2
(anonymous) @ NcActions.js:2
(anonymous) @ vue.runtime.esm.js:3113
wt @ vue.runtime.esm.js:3035
Promise.then (async)
Ot @ vue.runtime.esm.js:3060
Rt @ vue.runtime.esm.js:3125
(anonymous) @ vue.runtime.esm.js:4175
e.update @ vue.runtime.esm.js:3482
e.notify @ vue.runtime.esm.js:695
set @ vue.runtime.esm.js:941
Rn.Bn.set @ vue.runtime.esm.js:5311
$_applyShowEffect @ floating-vue.es.js:764
$_applyShow @ floating-vue.es.js:743
setTimeout (async)
$_scheduleShow @ floating-vue.es.js:709
show @ floating-vue.es.js:509
$_autoShowHide @ floating-vue.es.js:849
Ft @ vue.runtime.esm.js:2987
e.run @ vue.runtime.esm.js:3503
dn @ vue.runtime.esm.js:4090
(anonymous) @ vue.runtime.esm.js:3113
wt @ vue.runtime.esm.js:3035
Promise.then (async)
Ot @ vue.runtime.esm.js:3060
Rt @ vue.runtime.esm.js:3125
(anonymous) @ vue.runtime.esm.js:4175
e.update @ vue.runtime.esm.js:3482
e.notify @ vue.runtime.esm.js:695
set @ vue.runtime.esm.js:941
Rn.Bn.set @ vue.runtime.esm.js:5311
openMenu @ NcActions.js:2
Ft @ vue.runtime.esm.js:2987
n @ vue.runtime.esm.js:1785
Ft @ vue.runtime.esm.js:2987
e.$emit @ vue.runtime.esm.js:3685
show @ floating-vue.es.js:510
e @ floating-vue.es.js:875
```

**Comment by nickvergessen** [comment]
Other case in the log indicated the viewer app:
```
11:45:08.890 viewer-main.js:2 Uncaught 
setTimeout (async)
l @ core-common.js:2
E @ core-common.js:2
r @ core-common.js:2
activate @ core-common.js:2
(anonymous) @ core-common.js:2
u @ core-common.js:2
(anonymous) @ core-common.js:2
(anonymous) @ core-common.js:2
o @ core-common.js:2
s @ core-common.js:2
Promise.then (async)
o @ core-common.js:2
s @ core-common.js:2
(anonymous) @ core-common.js:2
(anonymous) @ core-common.js:2
useFocusTrap @ core-common.js:2
(anonymous) @ core-common.js:2
(anonymous) @ core-common.js:2
wt @ core-common.js:2
Promise.then (async)
Ot @ core-common.js:2
Rt @ core-common.js:2
(anonymous) @ core-common.js:2
e.update @ core-common.js:2
e.notify @ core-common.js:2
set @ core-common.js:2
Rn.Bn.set @ core-common.js:2
$_applyShowEffect @ core-common.js:2
$_applyShow @ core-common.js:2
setTimeout (async)
$_scheduleShow @ core-common.js:2
show @ core-common.js:2
$_autoShowHide @ core-common.js:2
Ft @ core-common.js:2
e.run @ core-common.js:2
dn @ core-common.js:2
(anonymous) @ core-common.js:2
wt @ core-common.js:2
Promise.then (async)
Ot @ core-common.js:2
Rt @ core-common.js:2
(anonymous) @ core-common.js:2
e.update @ core-common.js:2
e.notify @ core-common.js:2
set @ core-common.js:2
Rn.Bn.set @ core-common.js:2
openMenu @ core-common.js:2
Ft @ core-common.js:2
n @ core-common.js:2
Ft @ core-common.js:2
e.$emit @ core-common.js:2
show @ core-common.js:2
e @ core-common.js:2
```

**Comment by szaimen** [comment]
> Other case in the log indicated the viewer app:

yes, I though so that it is likely the viewer app...

**Comment by CarlKuhligk** [comment]
Can confirm with Brave Browser ([Version 1.46.144 Chromium: 108.0.5359.128 (Offizieller Build) (64-Bit)](https://brave.com/latest/))

Nextcloud version: 25.0.2 - 25.0.2.3

**Comment by szaimen** [comment]
Please try again with 25.0.3 which should be out by next week friday.

**Comment by timcouchoud** [comment]
Can confirm

Nextcloud 25.0.2 - Nextcloud AIO
Chrome Version 108.0.5359.124 (Official Build) (64-bit)
Ubuntu 22.04.1 LTS

**Comment by szaimen** [comment]
Hi, please try again with 25.0.3 as I said above.

**Comment by timcouchoud** [comment]
Hi @szaimen , could you confirm that 25.0.3 will be available this Friday 13th Jan OR 20th Jan ? Thanks!

**Comment by nextcloud-command** [comment]
This issue has been automatically marked as stale because it has not had recent activity and seems to be missing some essential information. It will be closed if no further activity occurs. Thank you for your contributions.

**Comment by skjnldsv** [comment]
Can confirmed, fixed with 25.0.4
